acme/http.go

```diff
@@ -49,12 +53,21 @@ func (t *retryTimer) backoff(ctx context.Context, r *http.Request, res *http.Res
 	}
 }
 
-func (c *Client) retryTimer() *retryTimer {
-	f := c.RetryBackoff
-	if f == nil {
-		f = defaultBackoff
+func (c *Client) newRetrier() *retrier {
```

Comment:
If you wanted to keep the convention being used before, you could just call this: 

`func (c *Client) retrier() *retrier`

No strong feelings here though.

Reply:
Also, thinking a struct name `retries` might be more appropriate, since this structure doesn't actually do any of the retrying, but would get a second opinion here